### PR TITLE
added inProgress to filter and show certificates with inProgress status

### PR DESCRIPTION
**Pull request by Tolik170**
## Description

Added inProgress to filter and show certificates with inProgress status

#### Screenshots

|         Updated          |
| :-----------------------: 
|image|

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅ All tests passed locally and linter has no warnings and errors
- [x] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/configs/statuses.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const statuses = {
     `Користувач буде автоматично розблокований через ${period} днів `,
   USER_INACTIVE_FOREVER_STATUS: 'Користувач деактивований назавжди',
   ACTIVE_STATUS: 'Активний',
+  IN_PROGRESS_STATUS: 'В процесі',
```
Comment (Pipuss) [inline src/configs/statuses.js:14]: Maybe "wait for payment" will sound better?

**Review by Pipuss — APPROVED** [review]

File: src/configs/statuses.js (hunk 1 above)
Comment (Tolik170) [reply src/configs/statuses.js:14]: At frontend it have "В Процесі", so i added the same text here.

Comment (Pipuss) [reply src/configs/statuses.js:14]: Basically, its not so important for admin part _)

**Review by MickhaelSh — DISMISSED** [review]

**Review by tarasSanotskyi — APPROVED** [review]

File: src/pages/certificates/hooks/use-certificates.js [hunk 2]
```diff
@@ -37,9 +37,9 @@ const transformDate = (date) => {
 const checkStatus = (certificate) => {
```
Comment (Pipuss) [inline src/pages/certificates/hooks/use-certificates.js:37]: Will something be returned from this function?

Comment (Tolik170) [reply src/pages/certificates/hooks/use-certificates.js:37]: it will return the string

**Review by Pipuss — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**